weather_mv/loader_pipeline/sinks.py

```diff
@@ -514,3 +516,24 @@ def open_dataset(uri: str,
         beam.metrics.Metrics.counter('Failure', 'ReadNetcdfData').inc()
         logger.error(f'Unable to open file {uri!r}: {e}')
         raise
+
+
+def get_file_time(element: t.Any) -> int:
+    try:
+        if element.startswith("gs://"):
+            bucket_name, file_name = element[5:].split("/", 1)
```

Comment:
Use `urlparse`; can do something like this: https://github.com/google/weather-tools/blob/main/weather_mv/loader_pipeline/util.py#L294

Reply:
Done